version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.7.1-SNAPSHOT"
+version in ThisBuild := "0.7.2-SNAPSHOT"
```

Comment:
*changes literally everything in the world*
*increments revision component of version*

:-)

Reply:
ahahaha, this is actually left over from the merge with master, but it's quite hilarious nonetheless. Should we think about 1.0.0? Maybe release this early as a milestone of 1.0.0?

Reply:
I'm all in favor of an early milestone of 1.0.0. We could be super-conservative and go with 0.8.0 if you prefer. Basically up to you.

This is definitely bringing cats-mtl more in line with what I would *expect* as a user from a 1.0 of such a library.